src/types.ts

```diff
@@ -234,7 +237,52 @@ type FeedbackResponseData = {
   url: string
 }
 
-type WebsiteResponseData = {
+type WebsiteNonDefaultTwitterHandle = {
+  subdomain: null | string
+  path: null | string
+  twitter_handle: string
+}
+
+type Website = {
   domain: string
   twitter_handle: null | string
+  non_default_twitter_handles: ReadonlyArray<WebsiteNonDefaultTwitterHandle>
+}
+
+type ParseDomainListed = {
```

Comment:
You are a true hero for adding all these types to `parse-domain`

Reply:
Yeah it's written with typescript, so not sure why those didn't import directly, but oh well